core-aam/index.html

```diff
@@ -4339,6 +4339,109 @@ <h4 id="role-map-searchbox"><code>searchbox</code></h4>
+                <td>
+                  <span class="property">Role: <code>ROLE_FOOTER</code></span
+                  ><br />
+                </td>
+              </tr>
+              <tr>
+                <th>
+                  <abbr title="macOS Accessibility Protocol">AX API</abbr><sup>[<a href="#ftn.note1">Note 1</a>]</sup>
+                </th>
+                <td>
+                  <span class="property">AXRole: <code>AXGroup</code></span
+                  ><br />
+                  <span class="property">AXSubrole: <code>&lt;nil&gt;</code></span
+                  ><br />
+                  <span class="property">AXRoleDescription: <code>sectionfooter</code></span>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <h4 id="role-map-sectionheader"><code>sectionheader</code></h4>
+          <table aria-labelledby="role-map-sectionheader">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="role-reference" href="#sectionheader"><code>sectionheader</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>Computed Role</th>
+                <td>
+                  <p><code>sectionheader</code></p>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property">Role: <code>ROLE_SYSTEM_GROUPING</code></span
+                  ><br />
+                  <span class="property">Object Attribute: <code>xml-roles:sectionheader</code></span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="User Interface Automation">UIA</abbr></th>
+                <td>
+                  <span class="property">Control Type: <code>Group</code></span
+                  ><br />
+                  <span class="property">Localized Control Type: <code>sectionheader</code></span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="Accessibility Toolkit">ATK</abbr>/<abbr title="Assistive Technology - Service Provider Interface">AT-SPI</abbr></th>
+                <td>
+                  <span class="property">Role: <code>ROLE_HEADER</code></span>
+                </td>
+              </tr>
+              <tr>
+                <th>
+                  <abbr title="macOS Accessibility Protocol">AX API</abbr><sup>[<a href="#ftn.note1">Note 1</a>]</sup>
+                </th>
+                <td>
+                  <span class="property">AXRole: <code>AXGroup</code></span
+                  ><br />
+                  <span class="property">AXSubrole: <code>&lt;nil&gt;</code></span
+                  ><br />
+                  <span class="property">AXRoleDescription: <code>sectionheader</code></span>
```

Comment:
Missed this first time around. AXRoleDescription is a user-presented string, so should include the space. Presumably the same is true for UIA's _**Localized**_ Control Type".

I've also reached out to the API owners on whether they want a new AXSubrole.

Reply:
thank you for catching this!  i've updated the instances